include/ibamr/IBFEMethod.h

```diff
@@ -765,20 +765,111 @@ class IBFEMethod : public IBStrategy
     int d_max_level_number;
     std::vector<libMesh::EquationSystems*> d_equation_systems;
 
+    /// Number of parts owned by the present object.
     const unsigned int d_num_parts = 1;
+
+    /// Number of parts owned by the present object.
```

Comment:
In the current design, each part has its own `FEDataManager` to manage interaction between the part and the background grid.

Reply:
I used essentially that text for the doc string.